### PR TITLE
Docs: Use schemeless URL for webfont

### PR DESCRIPTION
**Pull request by jxnblk**

### CODE REVIEW

File: docs/src/App.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ class App extends React.Component {
           <link
             key="webfont"
             rel="stylesheet"
-            href="http://fonts.googleapis.com/css?family=Montserrat:400,600|Roboto+Mono"
+            href="//fonts.googleapis.com/css?family=Montserrat:400,600|Roboto+Mono"
```
Comment (odmln) [inline docs/src/App.js]: @jxnblk Do we want to change the font weights to 500 & 700 while we're at it?

Comment (jxnblk) [reply docs/src/App.js]: I think we'll want to wait until v2 is ready since the font weights in the theme are still 400 and 600, right?

**Comment by jxnblk** [comment]
Merging this since it's a quick fix for the docs

File: docs/src/App.js (hunk 1 above)
Comment (odmln) [reply docs/src/App.js]: True, good call. We can hold off.

**Pull request merged**